src/apps/uclient/session.h

```diff
@@ -28,39 +30,36 @@
  * SUCH DAMAGE.
  */
 
-#ifndef __SESSION__
-#define __SESSION__
+#ifndef COTURN_UCLIENT_SESSION_H_3F24DB4E_3426_47FF_BC7F_4F3E1FB3094B
```

Comment:
Is this necessary in this PR?
Does it conflict with another define?

Reply:
Yes, this conflicts with another define in another 3rd party project that my own code uses.

Not namespacing the header guards of a header file with the name of the project is basically guaranteed to have a conflict with something somewhere.

The UUID is also really important, as it provides resilience against having multiple same named files with different contents, which is something that happens a lot with the version control system that my work uses (Perforce), due to the specific way perforce does branching and tagging.

Reply:
How can it conflict with a define in another 3rd party project if this is part of a code of a standalone binary tool of coturn?

Reply:
`coturn` is not only used in binary form?

I use the code from coturn in source form to compile it into other things.

Reply:
I removed the header guard change. It can be applied in a standalone PR some other time.